plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegateTest.java

```diff
@@ -186,8 +186,6 @@ public void testGenerateRunConfiguration() throws CoreException {
         .generateServerRunConfiguration(launchConfiguration, server);
     assertNull(config.getHost());
     assertEquals((Integer) LocalAppEngineServerBehaviour.DEFAULT_SERVER_PORT, config.getPort());
-    assertEquals(LocalAppEngineServerBehaviour.DEFAULT_ADMIN_HOST, config.getAdminHost());
-    assertEquals((Integer) LocalAppEngineServerBehaviour.DEFAULT_ADMIN_PORT, config.getAdminPort());
```

Comment:
Sounds like CT4IJ allows selecting either devappserver1 or 2. Would it make sense to provide an enum to `generateServerRunConfiguration()` with the particular desired devappserver flavour?